package.json

```diff
@@ -189,7 +189,7 @@
   },
   "engines": {
     "node": ">=14.16.1",
-    "npm": ">=6.0.0 <7.0.0"
+    "npm": ">=6.0.0"
```

Comment:
```➜ npm --version
7.7.6

 ➜ node --version              
v15.14.0
```

Works for me.

Reply:
F.Y.I. Same errors of static check occur in env using `./aio/develop/run-npm-on-container.sh`. In this env, versions are as follows:
node: 14.16.1
npm: 6.14.12

But e2e test and dashboard works properly.

Reply:
I could not find the way to fix it today. Have a nice weekend.

Reply:
You too, I'll try to investigate next week too.

Reply:
I've reverted it for now. Let's focus on LTS only.